controllers/StockApiController.php

```diff
@@ -322,7 +326,23 @@ public function ConsumeProductByBarcode(\Psr\Http\Message\ServerRequestInterface
 	{
 		try
 		{
-			$args['productId'] = $this->getStockService()->GetProductIdFromBarcode($args['barcode']);
+			if (Grocycode::Validate($barcode))
```

Comment:
Sorry, but have you tested this at least once? `$barcode` is undefined here...

Reply:
Sorry this was a push from the wrong repo locally